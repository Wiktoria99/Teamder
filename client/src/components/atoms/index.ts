export * from './BackButton';
export * from './CustomTextField';
export * from './CustomButton';
export * from './SideBarButton';
