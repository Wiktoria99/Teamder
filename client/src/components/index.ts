export * from "./atoms";
export * from "./atoms";
export * from "./molecules";
export * from "./organisms";
