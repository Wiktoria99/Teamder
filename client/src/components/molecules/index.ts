export * from './RegisterSteps';
export * from './TeamItem';
export * from './Suggestions';
export * from './Sidebar';
export * from './TeamList';
export * from './SuggestionItem';
export * from './SuggestionList';
export * from './NotificationItem';
export * from './NotificationList';
