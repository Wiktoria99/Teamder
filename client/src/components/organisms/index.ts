export * from './Layout';
export * from './FormWrapper';
export * from './MainWrapper';
export * from './BarWrapper';
