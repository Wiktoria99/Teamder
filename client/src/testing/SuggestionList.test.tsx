import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import { SuggestionItem, SuggestionList } from '@/components';
import { cleanup } from '@testing-library/react';

afterEach(cleanup);

const interest = {
  name: '',
  id: 1,
};

it('SuggestionItem renders without crashing', () => {
  shallow(<SuggestionItem interest={interest}/>);
});

it('SuggestionItem matches snapshot', () => {
  const tree = shallow(<SuggestionItem interest={interest}/>);
  expect(toJson(tree)).toMatchSnapshot();
});

it("accepts SuggestionItem props", () => {
  const wrapper = mount(<SuggestionItem interest={interest}/>);
  expect(wrapper.props().interest).toEqual(interest);
});

it('SuggestionList renders without crashing', () => {
  shallow(<SuggestionList/>);
});

it('SuggestionList matches snapshot', () => {
  const tree = shallow(<SuggestionList/>);
  expect(toJson(tree)).toMatchSnapshot();
});
