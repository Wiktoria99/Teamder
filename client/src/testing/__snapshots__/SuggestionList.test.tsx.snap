// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuggestionItem matches snapshot 1`] = `
<Styled(MuiBox)
  className="makeStyles-suggestionItemContainer-1"
>
  <a
    className="makeStyles-interestTitle-2"
    onClick={[Function]}
  >
    #
  </a>
  <p
    className="makeStyles-teamsNumber-3"
  >
    1
     Zespołów
  </p>
</Styled(MuiBox)>
`;

exports[`SuggestionList matches snapshot 1`] = `
<Styled(MuiBox)
  className="makeStyles-interestList-5"
>
  <Loading />
</Styled(MuiBox)>
`;
